Popins !!

SteveBourdin opened the pull request.

Laskewitz commented on power-apps/multiple-popin/README.md:
````diff
@@ -0,0 +1,1864 @@
+
+## Code
+
+``` YAML
````
I think we can just put a reference to the file here instead of adding the whole code in the readme. Might make the readme a bit too large if we keep it this way. 

You can remove the code and just add a link to the YAML file, like you did on line 22.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+                  var_popinConfirmation_No
+              )
+            Visible: =false
+            Width: =popinConfirmation_buttons.Width/4
```
You missed the ending of your code snippet by adding three backticks (```). The disclaimer below is now seen as part of the code snippet. When replacing the code snippet by a link, make sure to keep the disclaimer in the readme.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+If you want to use this Pop-In for the same screen for several cases, you can create a variable and then test it in the actions above.
+Exemple : 
+
+UpdateContext(
```
Please add three backticks (```) in front (before `UpdateContext(`) and behind (after ) ) your code snippet - that way people can easily copy the code.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+
+Exemple :
+
+UpdateContext(
```
Please add three backticks (```) in front (before `UpdateContext(` ) and behind (after ) ) your code snippet - that way people can easily copy the code.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+   - var_popinConfirmation_Show => boolean => if true, show the popin
+
+
+Exemple :
```
Please change Exemple in Example.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+
+
+If you want to use this Pop-In for the same screen for several cases, you can create a variable and then test it in the actions above.
+Exemple : 
```
Please change Exemple in Example.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+
+  - var_popinProgressBar_Current : Interger => Order of the current action
+
+Exemple : 
```
Please change Exemple in Example.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+
+As the actions progress, increment the variable var_popinProgressBar_Current 
+
+Exemple :
```
Please change Exemple in Example.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+
+Exemple : 
+
+UpdateContext(
```
Please add three backticks (```) in front (before `UpdateContext(`) and behind (after );; ) your code snippet - that way people can easily copy the code.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+
+# Use Pop-in Waiting :
+---------------------------
+Setting the var_popinLoading variable to true will display the loading window : UpdateContext({var_popinLoading: true});;
```
Please add three backticks (```) in front (before `UpdateContext(`) and behind (after ; ) your code snippet - that way people can easily copy the code.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+
+1. Paste the Yaml code on your screen
+
+2. Put the language in the OnVisible attribute of your screen : UpdateContext({var_popin_lang : "USA"}) (disponibilities : GBR, USA, FRA, ESP, PRT, ITA, CHN, DNK, DEU, NLD)
```
Please add three backticks (```) in front (before UpdateContext() and behind (after ) ) your code snippet - that way people can easily copy the code.

The part `(disponibilities : GBR, USA, FRA, ESP, PRT, ITA, CHN, DNK, DEU, NLD)` is invalid code.

Laskewitz commented on power-apps/multiple-popin/README.md:
```diff
@@ -0,0 +1,1864 @@
+
+Snippet|Author
+--------|---------
+Steve Bourdin | [GitHub](https://github.com/SteveBourdin) ([@steve-bourdin-ab998762](https://www.linkedin.com/in/steve-bourdin-ab998762/) )
```
Please change this to:

`Steve Bourdin | [GitHub](https://github.com/SteveBourdin) ([LinkedIn](https://www.linkedin.com/in/steve-bourdin-ab998762/) )`

Laskewitz reviewed (changes requested):
Hi @SteveBourdin - 

Thanks for your pull request! It looks like a good snippet - when I look at the image. While I was trying out your snippet, I just got a blank screen though.

I just pasted the code in the YAML on a screen and I used the following code for the OnVisible on the screen `UpdateContext({var_popin_lang : "USA"})` like what was in the instructions on how to use the snippet. Are there any steps missing?

<img width="2512" alt="image" src="https://github.com/user-attachments/assets/72ae3a15-a8ce-4ce4-8749-17822533bc35" />

Apart from that, I left a couple of comments in the review. Can you change those?

Thanks!

SteveBourdin commented:
Hi @Laskewitz  Thank you very much for this code review and the explanations. I have addressed all the points. For the default snippet, no pop-up appears, and we get a blank screen at first. The variables need to be populated in order to display a pop-up. I’ve added a YouTube video to make it more understandable.

Laskewitz reviewed (approved):
Great job @SteveBourdin!!! Merging!
